Visma.net/lib/DAta/LocationData.cs

```diff
@@ -0,0 +1,123 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Threading.Tasks;
+using ONIT.VismaNetApi.Models;
+
+namespace ONIT.VismaNetApi.Lib.Data
+{
+    public class LocationData : BaseCrudDataClass<Location>
```

Comment:
How about just skipping inheriting from BaseCrudDataClass?

Reply:
Would makes sense, though the implementation makes use of Add and Update in BaseCrudDataClass and we might want to keep the inheritance for code consistency but I'll let you decide on that.

Reply:
Has a second look at it and changed the inheritance from BaseCrudDataClass to BaseDataClass.